model/metrics/vcs.yaml

```
@@ -0,0 +1,150 @@
+groups:
+  - id: metric.vcs.repository.change.count
+    type: metric
+    metric_name: vcs.repository.change.count
+    brief: 'The number of changes (pull requests) in a repository, categorized by their state (either open or merged)'
+    instrument: gauge
+    unit: "{change}"
+    stability: experimental
+    #extends: metric_attributes.vcs
+    attributes:
+      - ref: vcs.change.state
+        requirement_level: required
+      - ref: vcs.repository.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_open
+    type: metric
+    metric_name: vcs.repository.change.time_open
+    brief: 'The amount of time a change (pull request) has been open'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.name
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_approval
+    type: metric
+    metric_name: vcs.repository.change.time_to_approval
+    brief: 'The amount of time it took a change (pull request) to go from open to approved'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.name
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.change.time_to_merge
+    type: metric
+    metric_name: vcs.repository.change.time_to_merge
+    brief: 'The amount of time it took a change (pull request) to go from open to merged'
+    instrument: gauge
+    unit: "s"
+    stability: experimental
+    attributes:
+      - ref: vcs.repository.name
+        requirement_level: required
+      - ref: vcs.repository.ref.name
+        requirement_level: required
+  - id: metric.vcs.repository.count
+    type: metric
+    metric_name: vcs.repository.count
+    brief: 'The number of repositories in an organization'
+    instrument: gauge
+    unit: "{repository}"
+    stability: experimental
+    attributes: []
+  - id: metric.vcs.repository.ref.count
+    type: metric
+    metric_name: vcs.repository.ref.count
+    brief: 'The number of refs of type branch in a repository'
```

Since we do have the `ref.type` attribute on this metric, should this say?
```suggestion
    brief: 'The number of refs of type branch or tag in a repository'
```

---

Done in 2da96a53e779912c7371c76184d18d4a268a3a37